app/collections/create-form.jsx

```diff
@@ -40,7 +40,7 @@ class CollectionsCreateForm extends React.Component {
     apiClient.type('collections').create(collection).save()
       .then((newCollection) => {
         this.name.value = '';
-        this.isPrivate.value = true;
+        this.isPrivate.value = null;
```

Comment:
We shouldn't change this from the original for the conversion. I left a comment because I didn't understand it, but that doesn't mean it should change. For the upcoming bugfixes and features that will be implementing, some time should be spent to understand it and if it's cruft, then it should be removed.